protobuf/query.proto

```diff
@@ -30,26 +30,23 @@ message Query {
     message Req {
         Options options = 1;
         oneof req {
-            Graql.Delete.Req delete_req = 100;
-            Graql.Define.Req define_req = 101;
-            Graql.Undefine.Req undefine_req = 102;
-            Graql.Match.Req match_req = 103;
-            Graql.Insert.Req insert_req = 104;
+            Delete.Req delete_req = 100;
+            Define.Req define_req = 101;
+            Undefine.Req undefine_req = 102;
+            Match.Req match_req = 103;
+            Insert.Req insert_req = 104;
```

Comment:
We additionally need:
```protobuf
MatchAggregate.Req match_aggregate_req = 105;
MatchGroup.Req match_group_req = 106;
MatchGroupAggregate.Req match_group_aggregate_req = 107;
```
And the matching responses should be added to `Res`, below

Reply:
Fixed in https://github.com/graknlabs/protocol/pull/100/commits/e6354fe87b04c8f37c5c81c8387d24c5fdcedfa4